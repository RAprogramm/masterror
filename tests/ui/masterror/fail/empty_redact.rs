use masterror::{AppCode, AppErrorKind, Masterror};

#[derive(Debug, Masterror)]
#[error("oops")]
#[masterror(code = AppCode::Internal, category = AppErrorKind::Internal, redact())]
struct EmptyRedact;

fn main() {}
