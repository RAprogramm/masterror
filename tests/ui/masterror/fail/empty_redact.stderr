error: redact(...) requires at least one option
 --> tests/ui/masterror/fail/empty_redact.rs:9:74
  |
9 | #[masterror(code = AppCode::Internal, category = AppErrorKind::Internal, redact())]
  |                                                                          ^^^^^^

warning: unused imports: `AppCode` and `AppErrorKind`
 --> tests/ui/masterror/fail/empty_redact.rs:5:17
  |
5 | use masterror::{AppCode, AppErrorKind, Masterror};
  |                 ^^^^^^^  ^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
